package-lock.json

```diff
@@ -15,7 +15,7 @@
 				"@feathersjs/errors": "^4.5.11",
 				"@feathersjs/express": "^4.5.11",
 				"@feathersjs/feathers": "^4.5.11",
-				"@feathersjs/socketio": "^4.5.11",
```

Comment:
running feathers v4 with some packages on v5 sounds messy and in my opinion can cause unexpected problems